compiler/passes/src/processing_async/mod.rs

```diff
@@ -54,6 +70,7 @@ impl Pass for ProcessingAsync {
             // checking again. That's because this pass introduces new `async function`s to the program.
             visitor.state.symbol_table = SymbolTable::default();
             SymbolTableCreation::do_pass((), state)?;
+            PathResolution::do_pass((), state)?;
```

Comment:
Should this be done before symbol table creation? To match the order in `compiler.rs`?